test/build/alpine/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:12.20.0-alpine
```

Comment:
I don't think it currently works in Alpine due to a missing `.so` library.

Reply:
This was just so the avado guys could debug. Not used by us.